src/components/header/header.js

```diff
@@ -0,0 +1,298 @@
+
+export default class HeaderComponent extends HTMLElement {
+    constructor() {
+        super();
+
+        const bodyElement = document.body;
+        bodyElement.style.margin = '0';
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `
+        
+        <header>
+            <div class="logo-bar">
+                <img class="greenwood-logo" src="/assets/greenwood-logo.svg" alt="Logo">
+            </div>
+
+            <div class="nav-bar">
+
+                <ul class="nav-bar-menu">
+                    <li class="nav-bar-menu-item">Docs</li>
+                    <li class="nav-bar-menu-item">Guides</li>
+                    <li class="nav-bar-menu-item">Blog</li>
+                </ul>
+
+                <div class="social-tray">
+                    <li class="social-icon">
+                        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If these are the SVGs that we already committed into the project, let me see if I can statically pull these in as text, otherwise it would be a shame to duplicate that content / code.

I'm also thinking that since only menu is interactive, that the header itself can probably be served statically and that we can create a mobile-menu component which can focus on just the interactive bits, let me try and play around with that.